benchmarks/tpch/schemas/psql_native/indexes.sql

```diff
@@ -14,3 +14,5 @@ CREATE INDEX IDX_LINEITEM_SHIPDATE ON LINEITEM (L_SHIPDATE);
 CREATE INDEX IDX_LINEITEM_COMMITDATE ON LINEITEM (L_COMMITDATE);
 CREATE INDEX IDX_LINEITEM_RECEIPTDATE ON LINEITEM (L_RECEIPTDATE);
 CREATE INDEX IDX_ORDERS_ORDERDATE ON ORDERS (O_ORDERDATE);
+
+CREATE STATISTICS lineitem_mcv (ndistinct) ON l_suppkey, l_partkey FROM lineitem; 
```

Comment:
The statistics name is a bit misleading, since you're not using `mcv` but rather `ndistinct`.
This is true for the other files as well.

Reply:
good point. fixed :)